### PR TITLE
modify create cocos project script

### PR DESCRIPTION
**Pull request by chuanweizhang2013**

### CODE REVIEW

**Comment by dumganhar** [comment]
This PR could not be merged automatically. Please pull the latest codes.

File: template/multi-platform-cpp/proj.android/project.properties [hunk 1]
```diff
@@ -10,4 +10,4 @@
 # Project target.
 target=android-10
 
-android.library.reference.1=../../../cocos/2d/platform/android/java
+android.library.reference.1=../cocos2d/cocos/2d/platform/android/java
```
Comment (minggo) [inline template/multi-platform-cpp/proj.android/project.properties:13]: I think it is a problem if generated project in the root of cocos2d-x.

File: tools/project-creator/module/ui.py [hunk 2]
```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/python
+#coding=utf-8
+"""****************************************************************************
+Copyright (c) 2010 cocos2d-x.org
```
Comment (minggo) [inline tools/project-creator/module/ui.py]: 2010 -> 2013?

File: tools/project-creator/module/core.py [hunk 3]
```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/python
+#coding=utf-8
+"""****************************************************************************
+Copyright (c) 2010 cocos2d-x.org
```
Comment (minggo) [inline tools/project-creator/module/core.py]: 2010 -> 2013?

File: template/multi-platform-cpp/proj.android/project.properties (hunk 1 above)
Comment (minggo) [inline template/multi-platform-cpp/proj.android/project.properties:13]: Ok, the script will copy engine codes, so it is not problem.

File: tools/project-creator/module/ui.py (hunk 2 above)
Comment (chuanweizhang2013) [inline tools/project-creator/module/ui.py]: has modified

File: tools/bindings-generator [hunk 4]
```diff
@@ -1 +1 @@
-Subproject commit 6e880d8843dd51fb5c4094f2672c7c846bb85401
+Subproject commit 1b34f5852fcf494499be7eb5ce2bada72a0b16a2
```
Comment (minggo) [inline tools/bindings-generator]: Don't update submodule.

File: cocos/scripting/auto-generated [hunk 5]
```diff
@@ -1 +1 @@
-Subproject commit 32f91b7f62106ddef2bccb0471e83982dddef761
+Subproject commit 56286dddefb8d5775ac508b371d0452c08bfd191
```
Comment (dumganhar) [inline cocos/scripting/auto-generated]: ？？

File: tools/bindings-generator [hunk 6]
```diff
@@ -1 +1 @@
-Subproject commit c842ccaa4b31121ac5f7be99ca64cafeb8d422c5
+Subproject commit 6e880d8843dd51fb5c4094f2672c7c846bb85401
```
Comment (dumganhar) [inline tools/bindings-generator]: ？？

File: tools/project-creator/linux.json [hunk 7]
```diff
@@ -1,8 +0,0 @@
-{
-    "rename":               [],
-    "remove":               [],
-    "replace_package_name": [],
-    "replace_project_name": [".cproject",
-                             ".project",
-                             "Makefile"]
```
Comment (dumganhar) [inline tools/project-creator/linux.json:7]: I think we should rename "CMakeLists.txt"

File: cocos/scripting/auto-generated [hunk 8]
```diff
@@ -1 +1 @@
-Subproject commit 32f91b7f62106ddef2bccb0471e83982dddef761
+Subproject commit 2d90634905755c2b0106d47df9f0eff8c462ee4d
```
Comment (minggo) [inline cocos/scripting/auto-generated]: Don't modify submodule.

File: template/multi-platform-cpp/proj.linux/build.sh [hunk 9]
```diff
@@ -1,59 +1,6 @@
...
-
-make -C $COCOS2DX20_TRUNK/CocosDenshion/proj.linux DEBUG=1
-check_make_result
-
-make -C $COCOS2DX20_TRUNK/extensions/proj.linux DEBUG=1
-check_make_result
-
-make DEBUG=1
-check_make_result
+mkdir build
```
Comment (minggo) [inline template/multi-platform-cpp/proj.linux/build.sh]: Why change form `755` to `644`?

File: tools/project-creator/linux.json (hunk 7 above)
Comment (chuanweizhang2013) [inline tools/project-creator/linux.json:7]: which place

File: template/multi-platform-cpp/proj.linux/build.sh (hunk 9 above)
Comment (chuanweizhang2013) [inline template/multi-platform-cpp/proj.linux/build.sh]: it's a mistake

**Comment by minggo** [comment]
There is an error when i run on ubuntu 13.10

```
minggo@minggo-Parallels-Virtual-Platform:~/cocos2d-x/tools/project-creator$ python ./create_project.pyw
Traceback (most recent call last):
  File "./create_project.pyw", line 39, in <module>
    from module.ui import createTkCocosDialog
  File "/home/minggo/cocos2d-x/tools/project-creator/module/ui.py", line 42, in <module>
    from Tkinter import *
  File "/usr/lib/python2.7/lib-tk/Tkinter.py", line 42, in <module>
    raise ImportError, str(msg) + ', please install the python-tk package'
ImportError: No module named _tkinter, please install the python-tk package
```

**Comment by minggo** [comment]
After apply the patch, now can compile on linux, but crashed when run the generated application.

**Comment by dumganhar** [comment]
Thanks. :+1:

**Pull request merged**